### DISCUSSION TITLE
How to overwrite the RootLayout in next.js 13 app dir

### DISCUSSION THREAD

**Discussion by MuttakinHasib** [vercel/next.js#45143] [Help] [answered]
I want to have a root layout that will be shared for the home, blog, and faq pages
and another layout for the dashboard or profile page, and this layout.js should replace the root layout ... how can I implement this? And what will be the folder structure?

@leerob @ijjk, please help me

**Comment by guitheengineer** [comment, marked as answer]
You can accomplish different root layouts using [route groups](https://beta.nextjs.org/docs/routing/defining-routes#route-groups)

![image](https://user-images.githubusercontent.com/49620737/213922519-a5fec86a-f845-439b-90c2-1f5b8a6b789d.png)


In your case it would work by putting home, blog and faq in a group with a root layout, then dashboard and profile on another group with a different layout.js.
